chapter_convolutional-modern/alexnet.md

```diff
@@ -195,18 +198,17 @@ By comparison, GPUs can consist of thousands of small processing elements (NIVID
 The details differ somewhat between NVIDIA, AMD, ARM and other chip vendors. While each core is relatively weak,
 running at about 1GHz clock frequency,
 it is the total number of such cores that makes GPUs orders of magnitude faster than CPUs.
-For instance, NVIDIA's recent Ampere generation offers over 300 TFLOPs per chip for specialized 16 bit precision (BFLOAT16) matrix-matrix multiplications, and up to 20 TFLOPs for more general-purpose floating point operations (FP32).
-At the same time, floating point performance of CPUs rarely exceeds 1 TFLOPs (AWS' Graviton 2, for instance, reaches 2 TFLOPs peak performance for 16 bit precision operations).
-The reason for why this is possible is actually quite simple:
+For instance, NVIDIA's recent Ampere A100 GPU offers over 300 TFLOPs per chip for specialized 16 bit precision (BFLOAT16) matrix-matrix multiplications, and up to 20 TFLOPs for more general-purpose floating point operations (FP32).
+At the same time, floating point performance of CPUs rarely exceeds 1 TFLOPs (Amazon's Graviton 3, for instance, reaches 2 TFLOPs peak performance for 16 bit precision operations, a number similar to the GPU performance of Apple's M1 processor). There are many reasons why GPUs are much faster than CPUs in terms of FLOPs. 
 First, power consumption tends to grow *quadratically* with clock frequency.
 Hence, for the power budget of a CPU core that runs 4 times faster (a typical number),
 you can use 16 GPU cores at $\frac{1}{4}$ the speed,
 which yields $16 \times \frac{1}{4} = 4$ times the performance.
-Furthermore, GPU cores are much simpler
+Secondly, GPU cores are much simpler
 (in fact, for a long time they were not even *able*
 to execute general-purpose code),
-which makes them more energy efficient.
-Last, many operations in deep learning require high memory bandwidth.
+which makes them more energy efficient. For instance, they tend not to support speculative evaluation, it typically isn't possible to program each processing element individually, and the caches per core tend to be much smaller. 
```

Comment:
What's the logical flow among:
"they tend not to support speculative evaluation", "it typically isn't possible to program each processing element individually", and "the caches per core tend to be much smaller"?